### PR TITLE
Resolved Issue #3 and created a function to return message from the j…

### PR DESCRIPTION
**Pull request by shreyvishal**
## Description
Created a function to return message from the json response feeded to it from the AI module

## Features covered
* https://github.com/beckn/beckn-action-bot/issues/3

## Changes made
* Added a new function in AI module for getting meaningful message

## Screenshots/recordings
Screenshot 2024-03-27 at 7 58 26 PM

## Testing
Provide empty JSON response, expected result is success is false, and then provide valid JSON response, expected result is a meaningful message

## Self checks

- [ checked ] Code follows coding style guidelines.
- [ checked ] All tests pass successfully.
- [ checked ] No new linting errors or warnings.
- [ checked ] Documentation is updated, if necessary.
- [ checked ] Dependencies are updated, if required.
- [ checked ] There are no merge conflicts.

### CODE REVIEW

File: config/openai.json [hunk 1]
```diff
@@ -1,32 +1,96 @@
...
         { "key": "search", "description": "Perform a search" },
-        { "key": "select", "description": "The instruction declares the customer's cart (or equivalent) created by selecting objects from the catalog. This is equivalent of adding items to cart (or equivalent) by selecting them." },
-        { "key": "init", "description": "Initialize an order by providing billing and/or shipping details." },
+        {
+            "key": "select",
+            "description": "The instruction declares the customer's cart (or equivalent) created by selecting objects from the catalog. This is equivalent of adding items to cart (or equivalent) by selecting them."
+        },
+        {
+            "key": "init",
+            "description": "Initialize an order by providing billing and/or shipping details."
```
Comment (mayurvir) [inline config/openai.json:10]: Why these changes?

File: config/openai.json [hunk 2]
```diff
@@ -1,32 +1,96 @@
...
+        {
+            "key": "on_init",
+            "description": "Sends the response of initializing a process"
+        },
+        {
+            "key": "on_confirm",
+            "description": "Response of confirming an action"
+        },
+        {
+            "key": "on_status",
```
Comment (mayurvir) [inline config/openai.json:37]: What are these changes for?

File: config/openai.json [hunk 3]
```diff
@@ -1,32 +1,96 @@
...
+            "description": "Response for providing a rating"
+        },
+        {
+            "key": "on_support",
+            "description": "Response for requesting a support"
+        }
     ],
     "SUPPORTED_DOMAINS": [
-        {"key": "uei:charging", "description": "Used for energy transactions"}
+        { "key": "uei:charging", "description": "Used for energy transactions" }
```
Comment (mayurvir) [inline config/openai.json:62]: Unnecessary space added.

File: config/openai.json [hunk 4]
```diff
@@ -1,32 +1,96 @@
...
-        { "role": "system", "content": "A typical order flow should be search > select > init > confirm."},
-        { "role": "system", "content": "Auto-generate uuid wherever required and should be a valid uuid such as ."},
-        { "role": "system", "content": "Use the context to identify the transaction id returned in search api response and use it in subsequent api calls."},
-        { "role": "system", "content": "Use the response of search request from assistant for filling transaction_id, bpp_id, bpp_uri in the context of all calls except `search`."},
-        { "role": "system", "content": "Use the response from assistant to select items from the list of items provided by the assistant."}
-
+        {
+            "role": "system",
+            "content": "Your job is to identify the endpoint, method and request body from the given schema, based on the last user input and return the extracted details in the following JSON structure : \n\n {'url':'', 'method':'', 'body':''}'"
```
Comment (mayurvir) [inline config/openai.json:67]: Why are these changes required?

File: config/openai.json [hunk 5]
```diff
@@ -1,32 +1,96 @@
...
+            "role": "system",
+            "content": "Use the response of search request from assistant for filling transaction_id, bpp_id, bpp_uri in the context of all calls except `search`."
+        },
+        {
+            "role": "system",
+            "content": "Use the response from assistant to select items from the list of items provided by the assistant."
+        }
     ],
-    "PRESETS" : {
+    "PRESETS": {
```
Comment (mayurvir) [inline config/openai.json:90]: Unnecessary formatting change

**Review by mayurvir — CHANGES_REQUESTED** [review]
There are a lot of formatting changes, its hard to review with so many changes in place. Can you create a fresh pr which has changes only for the task and no formatting changes to the rest of the code. The test cases also seem to fail and, so we need to find out what is causing those.

Please make sure the PR has minimal changes

File: config/openai.json (hunk 1 above)
Comment (shreyvishal) [reply config/openai.json:10]: Its due to the prettier formatting @mayurvir

File: config/openai.json (hunk 5 above)
Comment (shreyvishal) [reply config/openai.json:90]: Its due to the prettier formatting @mayurvir

File: config/openai.json (hunk 4 above)
Comment (shreyvishal) [reply config/openai.json:67]: Its due to the prettier formatting @mayurvir

File: config/openai.json (hunk 3 above)
Comment (shreyvishal) [reply config/openai.json:62]: Its due to the prettier formatting @mayurvir

File: config/openai.json (hunk 2 above)
Comment (shreyvishal) [reply config/openai.json:37]: Its due to the prettier formatting @mayurvir
